site/content/zh-CN/docs/installation/_index.md

```diff
@@ -3,70 +3,69 @@ title: "安装"
-  - [Feature gates for graduated or deprecated features](#feature-gates-for-graduated-or-deprecated-features)
-- [What's next](#whats-next)
+- [开始之前](#before-you-begin)
+- [安装正式版本](#install-a-released-version)
+  - [通过 kubectl 安装](#install-by-kubectl)
+  - [通过 Helm 安装](#install-by-helm)
+  - [为 Prometheus-Operator 添加指标采集](#add-metrics-scraping-for-prometheus-operator)
+  - [为可见性 API 添加 API 优先级和公平性配置](#add-api-priority-and-fairness-configuration-for-the-visibility-api)
+  - [卸载](#uninstall)
+- [安装自定义配置的正式版本](#install-a-custom-configured-released-version)
+- [安装最新开发版本](#install-the-latest-development-version)
+  - [卸载](#uninstall-1)
+- [从源代码构建和安装](#build-and-install-from-source)
+  - [为 Prometheus-Operator 添加指标采集](#add-metrics-scraping-for-prometheus-operator-1)
+  - [卸载](#uninstall-2)
+- [通过 Helm 安装](#install-via-helm)
+- [更改特性门控配置](#change-the-feature-gates-configuration)
+  - [Alpha 和 Beta 功能的特性门控](#feature-gates-for-alpha-and-beta-features)
+  - [已毕业或已弃用的功能的特性门控](#feature-gates-for-graduated-or-deprecated-features)
+- [接下来是什么](#whats-next)
 
 <!-- /toc -->
 
-## Before you begin
+## 开始之前 {#before-you-begin}
 
-Make sure the following conditions are met:
+确保满足以下条件：
 
-- A Kubernetes cluster with version 1.29 or newer is running. Learn how to [install the Kubernetes tools](https://kubernetes.io/docs/tasks/tools/).
-- The `SuspendJob` [feature gate][feature_gate] is enabled. In Kubernetes 1.22 or newer, the feature gate is enabled by default.
-- (Optional) The `JobMutableNodeSchedulingDirectives` [feature gate][feature_gate] (available in Kubernetes 1.22 or newer) is enabled.
-  In Kubernetes 1.23 or newer, the feature gate is enabled by default.
-- The kubectl command-line tool has communication with your cluster.
+- 运行版本为 1.29 或更新的 Kubernetes 集群。了解如何[安装 Kubernetes 工具](https://kubernetes.io/docs/tasks/tools/)。
+- 启用了 `SuspendJob` [特性门控][feature_gate]。在 Kubernetes 1.22 或更新版本中，该特性门控默认启用。
+- （可选）启用了 `JobMutableNodeSchedulingDirectives` [特性门控][feature_gate]（在 Kubernetes 1.22 或更新版本中可用）。
+  在 Kubernetes 1.23 或更新版本中，该特性门控默认启用。
+- kubectl 命令行工具已与您的集群通信。
 
-Kueue publishes [metrics](/docs/reference/metrics) to monitor its operators.
-You can scrape these metrics with Prometheus.
-Use [kube-prometheus](https://github.com/prometheus-operator/kube-prometheus)
-if you don't have your own monitoring system.
+Kueue 发布[指标](/docs/reference/metrics)以监控其控制器组件。
+您可以使用 Prometheus 采集这些指标。
+如果您没有自己的监控系统，请使用 [kube-prometheus](https://github.com/prometheus-operator/kube-prometheus)。
 
-The webhook server in kueue uses an internal cert management for provisioning certificates. If you want to use
-  a third-party one, e.g. [cert-manager](https://github.com/cert-manager/cert-manager), follow the [cert manage guide](/docs/tasks/manage/installation).
+Kueue 中的 Webhook 服务使用内部证书管理来配置证书。如果您想使用
+第三方证书，例如 [cert-manager](https://github.com/cert-manager/cert-manager)，请遵循[证书管理指南](/docs/tasks/manage/installation)。
```

Comment:
```suggestion
Kueue 中的 Webhook 服务使用内部证书管理来配置证书。如果您想使用第三方证书，例如
[cert-manager](https://github.com/cert-manager/cert-manager)，请遵循[证书管理指南](/docs/tasks/manage/installation)。
```
Try not to break the line between two Chinese characters, otherwise there will be a space in the middle of the rendered result.